### PR TITLE
Navbar router

### PR DESCRIPTION
**Pull request by MaggieSchuetz**
Issue: https://github.com/MaggieSchuetz/capstone-project/issues/5

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,34 +1,45 @@
...
+import { useNavigate } from 'react-router';
+
 import Header from './components/Header';
 import EntryList from './components/EntryList';
 import JournalEntries from './data/JournalEntries';
 import Form from './components/Form';
+import NavBar from './components/shared/NavBar';
 
 function App() {
+  const [entryEdit, setEntryEdit] = useState({
```
Comment (FNKMYR) [inline src/App.js:14]: Due to looking so similar, the naming of entryEdit and editEntry make it a little harder to read the code properly.

**Review by FNKMYR — COMMENTED** [review]
Curious if it has something to do with the edit function, since that on my to-do list for next week: 
Shouldn't you be able to reduce the 'date', 'title' & 'text' states into a single object state? 
Looks good overall!

File: src/App.js (hunk 1 above)
Comment (MaggieSchuetz) [reply src/App.js:14]: Thanks! Changed editEntry to editJournalEntry

**Review by F-Kirchhoff — APPROVED** [review]
Amazing 🦊

**Pull request merged**